android/src/main/java/com/dylanvann/fastimage/FastImageViewModule.java

```diff
@@ -33,6 +43,7 @@ public void preload(final ReadableArray sources) {
             public void run() {
                 for (int i = 0; i < sources.size(); i++) {
                     final ReadableMap source = sources.getMap(i);
+
```

Comment:
You should keep the author's original code style, so you can remove this empty line :)